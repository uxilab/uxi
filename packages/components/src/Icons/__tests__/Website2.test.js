import React from 'react';
import Website2 from '../Website2';

describe('Icon : <Website2 />', () => {
  it('match snapshot', () => {
    expect(shallow(<Website2 />)).toMatchSnapshot();
  });
});

