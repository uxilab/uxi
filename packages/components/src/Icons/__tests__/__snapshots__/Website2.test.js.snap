// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon : <Website2 /> match snapshot 1`] = `
<SvgIcon
  color="#6d6d71"
  hoverColor="#6d6d71"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  size={24}
  style={Object {}}
  viewBox="0 0 24 24"
>
  <svg
    focusable="false"
    height="24px"
    viewBox="0 0 16 16"
    width="24px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      data-name="Layer 2"
      id="Layer_2"
    >
      <g
        data-name="Layer 1"
        id="Layer_1-2"
      >
        <path
          d="M8,0a8,8,0,1,0,8,8A8,8,0,0,0,8,0ZM6.64,12.68H4.75a1.22,1.22,0,0,1-1.21-1.22V6.81h3.1Zm5.82-1.22a1.22,1.22,0,0,1-1.21,1.22H7.83V6.81h4.63Zm0-5.84H3.54V4.54A1.22,1.22,0,0,1,4.75,3.32h6.5a1.22,1.22,0,0,1,1.21,1.22Z"
        />
      </g>
    </g>
  </svg>
</SvgIcon>
`;
